CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### v0.3.2 - 2023-07-14
```

Comment:
This is a breaking change, so
```suggestion
### v0.4.0 - 2023-xx-xx
```

It won't be release today, but at some point later.